internal/binutils/binutils.go

```diff
@@ -47,19 +54,50 @@ type Binutils struct {
 	fast bool
 }
 
+// get returns the current representation for b, initializing it if necessary.
+func (bu *Binutils) get() *binrep {
+	bu.Mutex.Lock()
+	r := bu.rep
+	if r == nil {
+		r = &binrep{}
+		initTools(r, "")
+		bu.rep = r
+	}
+	bu.Mutex.Unlock()
+	return r
+}
+
+// update modifies the rep for b via the supplied function.
```

Comment:
nit: "for bu"

Reply:
Done.

Reply:
Yes. profile.Copy should also be fixed.  I think a separate PR since it is a completely separate code path?